pyproject.toml

```diff
@@ -34,7 +34,7 @@ dependencies = [
     "tweakwcs >=0.8.6",
     "spherical-geometry >= 1.2.22",
     "stsci.imagestats >= 1.6.3",
-    "drizzle >= 1.13.7",
+    "drizzle >= 1.14.0",
```

Comment:
This was needed as an outlier at the very edge of the image wasn't being picked up in drizzle <1.14.0.